File: Java/Basic Concepts .java
//2 Lesson

//The main Method
//Fill in the blank to declare a method called "hello()".
void hello();
//Which method prints text in a Java program?
System.out.println();
//Fill in the blanks to create a valid Java program.
class
 Apples {

    public static void 
main
(String[ ]args) {

    System.out.println("Hello, World!");

   }

}

//3 Lesson

//Single-line comments are created using:
// characters at the beginning of the line

//Make this text a multi-line comment.
/*
 some

* comment text

*/

//You can add a Java doc style comment by using:
/** and */ characters to wrap a comment.

//4 Lesson

//Which variable type would you use for a city name?
//String

//Drag and drop from the options below to have a valid Java program.

class Apples {

  public static void main(String[ ]args) {  
 name = "John";    
 age = 24;    
 height = 189.87;
  }
}

//5 Lesson


//Drag and drop from the options below to get user input.
import java.util.Scanner;
class test {
   public static void main(String[ ] args) {      
 Scanner sc = new Scanner(System.in);
      String st = sc.nextLine();
   }
}

//6 Lesson

//Fill in the blank to declare an integer variable and set its value to 5.
int var= 5;

//Fill in the blanks to print the sum of the two variables.
int x = 2; int y = 4;

int result = x + y;
System.out.println(result);

//What is the result of the following code?
//int x = 15; int y = 4;
//int result = x / y;
//System.out.println(result);

//3

//What value is stored in the result variable?
//int x = 8, y = 5;
//int result = x % y;

//3

//7 Lesson

//Fill in the blanks to print 11.

int a = 10;
++a;
System.out.println(a);

//What is the output of the following code?
//int x = 14; 
//System.out.println(x++);

//14

//Fill in the missing parts in the following code to print 13.
int x = 25; int y;
y = x - 12;

System.out.println(y);

//8 Lesson

//Drag and drop from the options below to print "Hello".
String var;
var = "Hello";
System.out.println(var);

//Which statement in regard to the char data type is true?

//'k' is a char

//9 Lesson

//Please type in a code to declare two variables of type int and print their sum using the sum variable.

int x = 4;

int y = 7;

int sum = x+y;
System.out.println(sum);

//In every Java program...

//...there must be a method called "main".

//Drag and drop from the options below to output the name:
String name;
name = "David";
System.out.println(name);

/*You need a program to convert days to seconds.
The given code takes the amount of days as input. Complete the code to convert it to seconds and output the result.

Sample Input:
12

Sample Output:
1036800*/

import java.util.Scanner;

public class Program {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int days = scanner.nextInt();
		int hours = 24;
		int minutes = 60;
		int seconds = 60;
		int result;
		result = days * hours * minutes * seconds;
		System.out.println(result);
		
	}
}

File: C#/Basic Concepts.cs
// 1 Lesson

//C# applications run:

//on the .NET Framework

//Which one is NOT part of the .NET Framework?

//Operation System

// 2 Lesson

//Which is a valid C# variable name?

//Bad_Var

//Fill in the blanks to declare a variable named num of type integer and assign 42 to it.

int num;

num = 42;

//Drag and drop the correct data types from the options below.
bool a = false;

double b = 4.22;

string c = "Hi";

int d = 11;

// 3 Lesson

//What is the name of the IDE used to create C# programs?

//Visual Studio

//Every console application in C# should contain:

//Main method

//Which type of application uses a text-only interface?

//Console Application

// 4 Lesson

//Drag and drop from the options below to display "Learning C#".

Console.WriteLine("Learning C#");

/*
What is the output of this code?
int a = 4;
int b = 2;
Console.Write(a);
Console.Write(b);
*/

//42

// 5 Lesson

//Drag and drop from the options below to take user input and store it in the temp variable:
string temp;

temp = Console.ReadLine();

//Drag and drop from the options below to make the following program work:
double n;

string x = "77";

n = Convert.ToDouble(x);

// 6 Lesson

/*
What is the output of this code?
int x = 8;
// x = 3;
Console.WriteLine(x);
*/

//8

//Fill in the blanks to make the text a comment.
/*
 Declaring an integer

  and printing it 
*/


int x = 42;

Console.WriteLine(x);

// 7 Lesson

//What is the type of the temp variable?
//var temp = 14.55;

//Double

/*
What is the output of this code?
var n1;
n1 = true;
Console.WriteLine(n1);
*/

//error

// 8 Lesson

//Fill in the blank to make the variable num a constant.

const int num=2;

// 9 Lesson

//Fill in the blanks to display the result of the multiplication of x and y.
int x = 42;

int y = 7;

int z = x*y;

Console.WriteLine(z);

/*
What is the output of this code?
int x = 16;
int y = 5;
Console.WriteLine(x/y);
*/

//3

//Which operator is used to determine the remainder?

//%

//Fill in the missing parentheses to have x equal 15.
int x = (2 + 3)* 3;

Console.WriteLine(x);


// 10 Lesson

//What is the alternative for x = x + 5?

//x += 5;

//Fill in the missing part of the following code to divide x by 3 using the shorthand division operator.
int x = 42;

x /= 3;

/*
What is the output of this code?
int x = 6;
x++;
Console.WriteLine(x);
*/

//7

//What's the difference between ++x and x++?
//Select All That Apply

//x++ uses x's value then increments it
//++x increments x's value before using it

//Fill in the missing operator to decrease the value of x by one.
int x = 42;

x--;

Console.WriteLine(x);


// 11 Lesson

//Rearrange the code to create a valid program.

static void Main(string[] args){
	string msg = "Hello";
	Console.WriteLine(msg);
}

//Drag and drop from the options below to display the value of x to the screen.
static void Main(string[] args)

{

   int x = (4 + 3) * 2;
   
   Console.WriteLine(x);

}

/*
What is the output of this code?
int a = 4;
int b = 6;
b = a++;
Console.WriteLine(++b);
*/

//5

//Fill in the blanks to declare two variables of type int and display their sum to the screen.
int x = 8;

int y = 15;

Console.WriteLine(x+y);

/*
What is the output of this code?
int x = 15;
int y = 6;
x %= y;
Console.WriteLine(x);
*/

//3


// 12 Lesson

/*

Area of a Circle


We are going to write a program that will calculate the area of ​​a circle.
The area enclosed by a circle of radius r is πr², where π (pi) is the constant ratio of the circumference of any circle to its diameter, and r is the radius.
The given program declares a constant pi variable with a value of 3.14.
Complete the program to take the radius as input, then calculate and output the area of the circle.

Sample Input
5

Sample Output
78.5

*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            const double pi = 3.14;
            double radius;
            radius=Convert.ToDouble(Console.ReadLine());
            var result = radius*radius*pi;
            Console.WriteLine(result);
            
        }
    }
}